dask_kubernetes/operator/controller/controller.py

```diff
@@ -284,7 +279,6 @@ async def daskworkergroup_create(spec, name, namespace, logger, **kwargs):
             name=spec["cluster"],
         )
         new_spec = dict(spec)
-        kopf.adopt(new_spec, owner=cluster)
```

Comment:
We are doing this intentionally here so that new worker groups that get created manually are also adopted and cascade deleted when the cluster is deleted. Please restore this line.